From d2691c2c17c31df054139700bafe06afddb7002b Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Tue, 7 Mar 2017 17:03:26 +0200
Subject: [PATCH 1554/5242] MLK-14266 ARM: imx: mach-imx7d: remove mxc_restart

commit  96592d703a0669eae377d2aca02e7bc8713f6bac from
https://source.codeaurora.org/external/imx/linux-imx.git

mxc_restart was incorrectly added back in commit c81df74845c74d14
("MLK-13869 ARM: imx7d: run pm init during intialization").

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/mach-imx7d.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-imx/mach-imx7d.c b/arch/arm/mach-imx/mach-imx7d.c
index 2e639e6..47bab33 100644
--- a/arch/arm/mach-imx/mach-imx7d.c
+++ b/arch/arm/mach-imx/mach-imx7d.c
@@ -172,5 +172,4 @@ static void __init imx7d_map_io(void)
 	.init_machine	= imx7d_init_machine,
 	.init_late	= imx7d_init_late,
 	.dt_compat	= imx7d_dt_compat,
-	.restart        = mxc_restart,
 MACHINE_END
-- 
1.7.9.5

